.travis.yml

```diff
@@ -1,11 +1,12 @@
+---
```

Comment:
is this a typo?

Reply:
no, `yaml` files can start with `---` in the first-line which generally signals the beginning of a new document.
It's optional, so I can remove it if you prefer